mongo-movies/routes/celebrity.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const router = express.Router();
+const Celebrity = require("../models/celebrity").Celebrity;
+
+/* GET celebrities listing. */
+router.get("/", function(req, res, next) {
+  Celebrity.find({}, (err, result) => {
+    if (err) {
+      console.log("ERRRRROORR", err);
+      return next(err);
+    } else {
+      const data = {
+        celebrities: result
+      };
+      res.render("celebrity/index", data);
+    }
+  });
+});
+
+router.get("/new", function(req, res, next) {
+  res.render("celebrity/new");
+});
+
+router.get("/:id", function(req, res, next) {
+  const id = req.params.id;
+  Celebrity.findById(id, (err, result) => {
+    if (err) {
+      console.log("ERRRRROORR", err);
+      return next(err);
+    } else {
+      const data = {
+        celebrities: result
+      };
+      res.render("celebrity/show", data);
+      console.log(data);
+    }
+  });
+});
+
+router.post("/", (req, res, next) => {
+  const celebrityInfo = {
+    name: req.body.name,
+    occupation: req.body.occupation,
+    catchPhrase: req.body.catchPhrase
+  };
+
+  const newCelebrity = new Celebrity(celebrityInfo);
+
+  newCelebrity.save(err => {
+    if (err) {
+      return render("celebrity/new");
+    }
+    return res.redirect("../");
+  });
+});
+
+router.post("/", (req, res, next) => {
```

Comment:
You can't have two routes with the same verb (post in this case) and the same path ("/" in this case). The second one is never going to be reached.